com.cubrid.common.ui/src/com/cubrid/common/ui/er/editor/ERSchemaToolBar.java

```diff
@@ -409,15 +411,17 @@ private String getDBLabel() {
 	private Composite createAutoLayoutComp() {
 		Composite comp = new Composite(this, SWT.NONE);
 		final GridLayout gdLayout = new GridLayout(1, false);
-		gdLayout.marginHeight = 1;
-		gdLayout.marginWidth = 1;
-		gdLayout.horizontalSpacing = 2;
-		gdLayout.verticalSpacing = 0;
+		final GridData gridData = new GridData(SWT.FILL, SWT.FILL, true, true);
+		
 		comp.setLayout(gdLayout);
-		comp.setLayoutData(new GridData(SWT.FILL, SWT.FILL, true, false));
-
-		final Button autoLayoutButton = new Button(comp, SWT.PUSH | SWT.CENTER);
-		autoLayoutButton.setLayoutData(CommonUITool.createGridData(1, 1, 98, 20));
+		comp.setLayoutData(gridData);
+
+		final Button autoLayoutButton = new Button(comp, SWT.PUSH | SWT.CENTER | SWT.FILL);
+		Font font = autoLayoutButton.getFont();
+		String fontName = font.getFontData()[0].name;
```

Comment:
I found my mistake. 
`FontData.name` field is platform dependent. 
As a result, `FontData.name` couldn't use in Windows platform.
So we should change `name` field to `getName()` method like below.
`String fontName = font.getFontData()[0].getName();`